index.html

```diff
@@ -0,0 +1,292 @@
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Join</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">News</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="first-title">
+      <ul class="title-black">
+        <li>
+          <a href="#" class=""><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <nav class="second-title">
+      <div class="logo">
+        <a href="index.html"
+          ><img
+            class="logo-img"
+            src="https://www.arabnews.com/sites/default/files/main-image/2019/10/04/1783706-1871065082.jpg"
+            alt="riyadh season logo"
+        /></a>
+      </div>
+      <ul class="title-white">
+        <li>
+          <a href="about/about.html" class="nav-link text-muted">About</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Program</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Sponser</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Join</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">News</a>
+        </li>
+      </ul>
+      <div class="pt-1">
+        <a href="#" class="title-red">CC Campaign</a>
+      </div>
+    </nav>
+          <h2 class="display-2 header ps-3 mt-5">"Hello! Sharing World"</h2>
+      <h1 class="display-1 fw-bold header ps-3 pe-3">
+        Join our unique annual <br />
+        festival in the capital <br />
+        city of Saudia Arabia, Riyadh
+      </h1>
+      <p class="intro p-3 border border-4 m-3 border-white">
+        A jouful festival where people from around the world will experiance
+        different events. Where each event is far more unique than the other.
+      </p>
+      <p class="display-3 fw-bold text-muted pb-5 ps-3">
+        2022.3.21(FRI) ~ 22(SAT)
+      </p>
+    </header>
+
+    <!-- Main Program -->
+
+    <section id="home">
+      <h3 class="text-center text-white pt-5">Main Program</h3>
+      <hr style="width: 10%; height: 3px; color: red; margin-left: 45%" />
+      <div class="container-lg text-white mt-5">
+
+        <div class="row m-1 mt-2 align-items-center main-background">
+          <div class="col">
+            <i class="fas fa-ticket-alt"></i>
+          </div>
+          <div class="col text">
+            <p class="main-text">Tickets</p>
+          </div>
+          <div class="col-6">
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing & opening.
+            </p>
+          </div>
+        </div>
```

Comment:
- [ ] It would be nice if the tag of the cards in the main program are changed to `<article>` tags. This helps in using more semantic html code and improves its readability